client/api.go

```diff
@@ -74,23 +75,41 @@ func challengeForPKeyPassword() string {
 	return password
 }
 
-func privateKey(pkeyFilePath string) *rsa.PrivateKey {
+func privateKey(pkeyFilePath string) (*rsa.PrivateKey, error) {
```

Comment:
Feels like this is oracle specific - could it be moved int oracle_provider.go?

Reply:
Sure!